CDTDatastore/CDTReplicator/CDTPullReplication.m

```diff
@@ -65,17 +65,8 @@ - (instancetype)copyWithZone:(NSZone *)zone
 
 - (NSString *)description
 {
-    NSMutableDictionary *dictionary = [[NSMutableDictionary alloc] init];
-    
-    if ([CDTAbstractReplication validateOptionalHeaders:self.optionalHeaders error:nil]) {
-        dictionary[@"headers"] = self.optionalHeaders;
-        dictionary[@"interceptors"] = self.httpInterceptors;
-    } else {
-        return nil;
-    }
-    
-    dictionary[@"source"] = TDCleanURLtoString(self.source);
-    return [NSString stringWithFormat:@"%@: %@", [self class], dictionary];
+    return [NSString stringWithFormat:@"source: %@, target: %@, headers: %@, interceptors: %@, filter: %@, query_params: %@",
```

Comment:
You should also add `[self class]` to the output so a user can tell whether it's pull or push